README.md

```diff
@@ -0,0 +1,71 @@
+# Kiwix Apple Custom Apps
+
+Kiwix Apple custom apps are iOS/macOS apps running [Kiwix for
+Apple](https://github.com/kiwix/apple) against a
+pre-configured ZIM file.
+
+This project contains data and scripts needed to create specific
+ custom Kiwix Apple apps.
+
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+
+## Custom app folder
+
+[In the repository](https://github.com/kiwix/kiwix-apple-custom),
+each custom configuration is isolated in a so called custom app
+folder. If you need to create a new one for a new custom app, you can
+do that easily using Github UI. Go to
+https://github.com/kiwix/kiwix-apple-custom/new/main, put the name
+of your app (no space, lowercase only) and add a slash at the end.
+
+## info.json file
+
+The configuration of the custom app is handled using the `info.json`
+file which **is required to be placed in the custom app folder**. Take example on an already
+existing one if you need to create a new custom app. The most
+important fields are:
+- `about_app_url` - this is an external link that is placed in the "About section" of the application. Currently it is not supporting slashes, and ``https://`` is prepended to the value provided here. (Eg. given "about_app_url": "www.dwds.de", the final link in the app will be: "https://www.dwds.de")
```

Comment:
We should change this: full URL with should be given in the `info.json` or do we have a specific reason to keep this behaviour?

Reply:
I've created an issue for this, explaining why we have it like this, and how it can be fixed: https://github.com/orgs/kiwix/projects/18?pane=issue&itemId=48736879